setup.py

```diff
@@ -45,7 +45,7 @@ def run_tests(self):
 
 setup(
     name='invideoquiz-xblock',
-    version='0.2.0',
+    version='0.2.1',
```

Comment:
Why did we bump the minor version to x.2.x?

Per SemVer [1], the 3rd "variable" is the "patch" number, used to indicate a bugfix.
The "minor" number, what went from 1 to 2, is for backwards compatible feature changes.

I suspect this should just be `0.1.4`.

[1] http://semver.org/